lms/templates/dashboard/_dashboard_credit_info.html

```diff
@@ -17,69 +20,53 @@
             <p class="message-copy credit-request-pending-msg">
-            ${_("Thanks for your payment! We are currently processing your course credit. You'll see a message here when the transaction is complete. For more information, see {provider_link}.").format(
-                    provider_link=u'<a href="{provider_url}">{provider_name}</a>'.format(
-                        provider_url=credit_status["provider_status_url"],
-                        provider_name=credit_status["provider_name"],
+                ${_("Thank you. Your credit is processing. Please see {link} for more information.").format(
+                        link=provider_link,
                     )
-                )
-            }
+                }
             </p>
+            <button class="btn credit-btn pending-credit-btn" data-course-key="${credit_status['course_key']}" data-user="${user.username}" data-provider="${credit_status['provider_id']}">${_("Learn more")}</button>
         % elif credit_status["request_status"] is None:
             <p class="message-copy credit-request-pending-msg">
-            ${_("Thank you for your payment! To receive your course credit, go to the {provider_link} website. You may have to complete some additional steps depending on the institution").format(
-                    provider_link=u'<a href="{provider_url}">{provider_name}</a>'.format(
-                        provider_url=credit_status["provider_status_url"],
-                        provider_name=credit_status["provider_name"],
+                ${_("Thank you for your purchase. Please proceed to {link} to finalize your credit.").format(
+                        link=provider_link,
                     )
-                )
-            }
+                }
             </p>
+            <button class="btn credit-btn pending-credit-btn" data-course-key="${credit_status['course_key']}" data-user="${user.username}" data-provider="${credit_status['provider_id']}">${_("Finalize credit")}</button>
         % endif
 
-        <button class="btn credit-btn pending-credit-btn" data-course-key="${credit_status['course_key']}" data-user="${user.username}" data-provider="${credit_status['provider_id']}">${_("Complete Credit")}</button>
         <form id="credit-pending-form"> </form>
     % elif credit_status["request_status"] == "approved" and not credit_status["error"] :
         <p class="message-copy credit-request-approved-msg">
-        ${_("Congratulations - you have received credit for this course! For more information, see {provider_link}.").format(
-                provider_link=u'<a href="{provider_url}">{provider_name}</a>'.format(
-                    provider_url=credit_status["provider_status_url"],
-                    provider_name=credit_status["provider_name"],
+            ${_("Your credit has been processed and approved. {congrats}. Please see {link} for more information.").format(
+                    congrats="<b>Congratulations</b>",
```

Comment:
This text is not internationalized! And it's pulled out of the sentence! You can't do this!